pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,134 @@
+{ config
+, pkgs
+, fetchurl
+, atspiSupport ? true, at_spi2_core ? null
+, gtk3
+, gnome3
+, libcanberra_gtk3
+, libudev
+, hunspell
+, isocodes
+, pkgconfig
+, xorg
+, libxkbcommon
+, python3
+, python35Packages
```

Comment:
This is wrong. Either pass in `pythonX` and use `pythonX.pkgs` or pass in `pythonXPackages`. If you pass in both you have the risk they're not the same set. This is exactly what happens here, because `python3` is an alias `python36`.